playbooks/itsm_inventory.yaml

```diff
@@ -0,0 +1,171 @@
+- name: Inventory
+  hosts: |-
+    {%- if nfc_pb_host is defined -%}
+      {{ nfc_pb_host }}
+    {%- else -%}
+      all
+    {%- endif %}
+  become: true
+
+
+
+  tasks:
+
+    - name: Inventory host
+      block:
+    
+      - name: Test Packages
+        ansible.builtin.package_facts:
+          manager: auto
+        become: true
+
+
+
+      - name: Inventory Details
+        ansible.builtin.set_fact:
+          details: {
+            "name": "{{ ansible_hostname }}",
+            "serial_number": "{{ ansible_product_serial }}",
+            "uuid": "{{ ansible_product_uuid }}"
+          }
+
+
+      - name: Inventory Software [a-k]
+        ansible.builtin.set_fact:
+          cacheable: false
+          software: "{{ software | default([]) + [{
+              'name': package.value[0].name,
+              'category': package.value[0].category | default(''),
+              'version': package.value[0].version
+            }] }}"
+        # no_log: true
+        loop: "{{ ansible_facts.packages | dict2items() }}"
+        loop_control:
+          loop_var: package
+          label: "{{ package.key }}"
+        when: >
+          package.value[0].name | regex_search("^[a-k]")
+
+
+#     - name: Inventory Software [l]
```

Comment:
```suggestion
#
# This task has been commented out due to excessive memory usage (>5GB) for this playbook to complete
#
#     - name: Inventory Software [l]
```

also raise a ticket in this repo for this problem.

Reply:
ticket #19 raised

Reply:
add a link to the ticket above the comment.

Reply:
linked